Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -430,31 +372,46 @@ private bool ValidateRequest(string remoteIp, bool isLocal)
         /// <returns>True if the request is valid, or false if the request is not valid and an HTTPS redirect is required.</returns>
         private bool ValidateSsl(string remoteIp, string urlString)
         {
-            if (_config.Configuration.RequireHttps && _appHost.ListenWithHttps)
+            if (_config.Configuration.RequireHttps
+                && _appHost.ListenWithHttps
+                && !urlString.Contains("https://", StringComparison.OrdinalIgnoreCase))
```

Comment:
Noticed this while fixing the merge conflict. Should this be `StartsWith` instead of `Contains`?

Reply:
Well, it should, but I highly doubt we'd face a URL with `https://` not at the start, or that such URL would really cause any trouble... I'm more bothered with `return true` at the end of this function. But that seems to be an issue for a subsequent PR.

Reply:
If the URL is HTTP, this will scan the entire string looking for `https://` instead of just scanning the first five characters. But yeah, just a small performance thing